sskittenapi/catapi/catapi/events/event_handlers.py

```diff
@@ -37,6 +39,17 @@ def handle_cat_created(data: dto.JSON) -> None:
 
     logger.info(f"[{event_id}] Cat {cat_id} has been created")
     # TODO: Handle the async postprocessing of a created Cat here.
+    
+    cat_partial_update = dto.PartialUpdateCat(url=["http://placekitten.com/200/300"])
```

Comment:
@driek-vhees, I was not sure how to import/download the URL from that `placekitten` website, that's why I add this URL as static.
Your kind suggestions are open.

Reply:
@SShakeeb I think that is fine.